app/src/main/java/com/mk/tasky/agenda/home/presentation/HomeScreen.kt

```diff
@@ -66,13 +67,33 @@ fun HomeScreen(
                         title = it.title,
                         description = it.description,
                         color = Light,
-                        onOptionsClick = { /*TODO*/ },
+                        onOptionsClick = {
+                            viewModel.onEvent(HomeEvent.OnItemOptionsClick(it.id!!))
+                        },
                         startDatetime = it.dateTime
                     )
                     Spacer(modifier = Modifier.height(16.dp))
                 }
             }
         }
+        Box(
+            modifier = Modifier.fillMaxSize(),
+            contentAlignment = Alignment.CenterEnd
+        ) { // TODO: Make it so alignment changes based on position in screen
+            val itemOptions = HomeItemOptions.values()
+            TaskyDropdown(
+                items = itemOptions.map { it.type.asString(context) },
```

Comment:
This will execute a loop on every recomposition and not only when itemOptions changes (which it never does). Better use remember here.

Reply:
Will add remember. Thanks!